src/AliasUnit.cpp

```diff
@@ -232,8 +232,7 @@ int AliasUnit::getNewID()
 bool AliasUnit::processDataStream(const QString& data)
 {
     TLuaInterpreter* Lua = mpHost->getLuaInterpreter();
-    QString lua_command_string = "command";
-    Lua->set_lua_string(lua_command_string, data);
+    Lua->set_lua_string(QStringLiteral("command"), data);
```

Comment:
No because there is no `QLatin1String` overload for `set_lua_string` and `QLatin1String` will allocate at runtime while `QStringLiteral` allocates at compile time. See https://github.com/KDE/clazy/blob/master/src/checks/level2/README-qstring-allocations.md